test_outputs/struct_marked_non_exhaustive.output.ron

```diff
@@ -1,41 +1,59 @@
-        "struct_type": String("tuple"),
-        "path": List([
-            String("test_crates"),
-            String("test_cases"),
-            String("non_exhaustive"),
-            String("TupleStruct"),
-        ]),
-        "span_filename": String("src/test_cases/non_exhaustive.rs"),
-        "span_begin_line": Uint64(23),
-        "visibility_limit": String("public"),
-    },
-    {
-        "name": String("ExternallyConstructibleStruct"),
-        "struct_type": String("plain"),
-        "path": List([
-            String("test_crates"),
-            String("test_cases"),
-            String("non_exhaustive"),
-            String("ExternallyConstructibleStruct"),
-        ]),
-        "span_filename": String("src/test_cases/non_exhaustive.rs"),
-        "span_begin_line": Uint64(32),
-        "visibility_limit": String("public"),
-    }
-]
+{
+    "./test_crates/non_exhaustive/": [
+        {
+            "name": String("UnitStruct"),
+            "path": List([
+                String("non_exhaustive"),
+                String("UnitStruct"),
+            ]),
+            "span_begin_line": Uint64(12),
+            "span_filename": String("src/lib.rs"),
+            "struct_type": String("unit"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "name": String("TupleStruct"),
+            "path": List([
+                String("non_exhaustive"),
+                String("TupleStruct"),
+            ]),
+            "span_begin_line": Uint64(15),
+            "span_filename": String("src/lib.rs"),
+            "struct_type": String("tuple"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "name": String("ExternallyConstructibleStruct"),
+            "path": List([
+                String("non_exhaustive"),
+                String("ExternallyConstructibleStruct"),
+            ]),
+            "span_begin_line": Uint64(18),
+            "span_filename": String("src/lib.rs"),
+            "struct_type": String("plain"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "name": String("NonExternallyConstructibleTupleStruct"),
+            "path": List([
+                String("non_exhaustive"),
+                String("NonExternallyConstructibleTupleStruct"),
+            ]),
+            "span_begin_line": Uint64(25),
+            "span_filename": String("src/lib.rs"),
+            "struct_type": String("tuple"),
+            "visibility_limit": String("public"),
+        },
+        {
+            "name": String("NonExternallyConstructibleStruct"),
+            "path": List([
+                String("non_exhaustive"),
+                String("NonExternallyConstructibleStruct"),
+            ]),
+            "span_begin_line": Uint64(28),
+            "span_filename": String("src/lib.rs"),
+            "struct_type": String("plain"),
+            "visibility_limit": String("public"),
+        },
```

Comment:
These two results are unexpected. Adding `#[non_exhaustive]` on structs that were already not externally constructible is not breaking. I'm not sure why they are being incorrectly flagged now.

Reply:
Resolved in #222.